twentytwenty-blocks/twentytwenty-styles/style.css

```diff
@@ -2736,21 +2736,21 @@ h2.entry-title {
 /* CUSTOM COLORS */
 
 :root .has-accent-color {
```

Comment:
Is the `:root` selector still needed? I noticed that in #45 it wasn't used and still worked.

Reply:
Yeah, these don't get used at all currently — they're overridden by rules assigned to `.editor-styles-wrapper` instead.